runtime-management/runtime-management-impl/src/test/java/io/cloudslang/runtime/impl/python/PythonExecutorTest.java

```diff
@@ -338,10 +339,14 @@ public PythonExecutorConfigurationDataService pythonExecutorConfigurationDataSer
             return PythonExecutorDetails::new;
         }
 
+        @Bean(name = "pythonExecutorCommunicationService")
```

Comment:
You have two beans for PythonExecutorCommunicationServiceImpl. You can either have it here in the configuration class or use the @Service annotation, never do both.